rules/TypeDeclaration/Rector/FunctionLike/AddParamTypeSplFixedArrayRector.php

```diff
@@ -121,9 +124,12 @@ public function refactor(Node $node): ?Node
                 $param,
                 $paramName
             );
+
+            $hasChanged = true;
         }
 
-        if ($functionLikePhpDocInfo->hasChanged()) {
+        if ($hasChanged) {
+            $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($node);
```

Comment:
this should not needed, as already called while on `$this->phpDocTypeChanger->changeParamType()` above:

https://github.com/rectorphp/rector-src/blob/596bbc98a25903c525914b4d16ea5f9c76b500fb/packages/BetterPhpDocParser/PhpDocManipulator/PhpDocTypeChanger.php#L96

Reply:
@samsonasik I won't have time tonight to work on this, but I don't want it to be a blocker.
 Could you take this PR over and push the merge when its green? :)